src/Connection/PostObjects.php

```diff
@@ -362,12 +362,18 @@ public static function get_connection_config( $graphql_object, $args = [] ) {
 			unset( $connection_args['stati'] );
 		}
 
+		$from_field_name = lcfirst( $graphql_object->graphql_plural_name );
+
+		if ( $graphql_object->graphql_single_name === $graphql_object->graphql_plural_name ) {
```

Comment:
Perhaps we can _also_ support this pattern if `graphql_plural_name` is not set. 

Like if I do: 

```php
register_post_type( 'test', [
  'show_in_graphql' => true,
  'graphql_single_name' => 'test'
]);
```
Perhaps the lack of `graphql_plural_name` should also lead to `test` and `allTest` in the Schema with no errors 🤔

Reply:
We still need _something_ set for `graphql_plural_name` for the sake of generating error messages and for simpler Type->Type connections. Checking for an unset or empty plural name in all of these case could get messy and cause a lot of branching. Maybe we could add a filter on `register_post_type()` and `register_taxonomy()` args that would set `$args['graphql_plural_name'] = $args['graphql_single_name']` if plural is not provided? I'll give that a try in a follow up PR.